Snackbar/Snackbar.jsx

```diff
@@ -1,13 +1,21 @@
 import { h } from "preact";
 import MaterialComponent from "../MaterialComponent";
 import { MDCSnackbar } from "@material/snackbar/";
+
+function shallowDiffers(a, b) {
+  for (let i in a) if (!(i in b)) return true;
+  for (let i in b) if (a[i] !== b[i]) return true;
+  return false;
+}
+
 /**
  * dismissesOnAction = true
  */
 export default class Snackbar extends MaterialComponent {
   constructor() {
     super();
     this.componentName = "snackbar";
+    this.isPureReactComponent = true;
```

Comment:
this isn't required right?

Reply:
Probably not, I copied from `preact-compat/PureComponent` class